main.py

```diff
@@ -1,16 +1,21 @@
 import asyncio
+import os
 from tycho.app import create_app
 from tycho.models.config import Config
 
+DB_URI = os.environ.get(
+    "TYCHO_MONGODB_URI",
+    "mongodb://mongo:27017/?maxPoolSize=1&w=1"
```

Comment:
might want to make all these configurable. i.e. port number, max pool size and w

Reply:
mongodb makes all the parameters configurable via the URI: https://docs.mongodb.com/manual/reference/connection-string/

Thus, it's possible to modify the mongodb connect behavior without touching any application code. I don't think there's a lot of value in configuring the default URI, as it expects the dns name "mongo" which really only works with docker-compose.